crates/ruma-common/src/events/room/message.rs

```diff
@@ -104,6 +109,12 @@ impl RoomMessageEventContent {
     /// the previous message, for which the room ID is required. If you want to reply to an
     /// [`OriginalSyncRoomMessageEvent`], you have to convert it first by calling
     /// [`.into_full_event()`][crate::events::OriginalSyncMessageLikeEvent::into_full_event].
+    ///
+    /// It is recommended to enable the `sanitize` feature when using this method as this will
```

Comment:
There's quite a bit of documentation duplication here, maybe create one or more markdown files again and include them? (also in the existing text)